### ISSUE TITLE
Bintray is no longer accessible

### ISSUE THREAD

**Issue by augusto-cactus** [kermitt2/grobid-example#11]
Hi, it looks like bintray repository has been discontinued, is there any other place we could get grobid as a maven dependency? 

Best regards

**Comment by kermitt2** [comment]
Hello @augusto-cactus ! Thanks for the issue. 

Normally bintray is staying read-only? It would then not affect getting the current artifacts as maven dependencies. Otherwise, I guess I will need to upload the dependencies manually on maven. 
For sure the problem will be for the next release, the publication script will have to be updated to use maven like in the old days.

**Comment by augusto-cactus** [comment]
Hi @kermitt2 thanks for the quick answer - I am not sure about that at least I can't access the repository when we try to build a project that uses grobid. Also if I follow the link https://dl.bintray.com/rookies/maven/org/grobid/grobid-core/0.6.1/grobid-core-0.6.1.pom I get a forbidden.

**Comment by kermitt2** [comment]
You're right it seems bintray is unavailable, despite the "read only" announcement (initially until 2022, but then "indefinitely" https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/). Sorry for this, I was too optimistic. I'll try to find a quick solution. 
In between, I am afraid there's nothing but a local grobid install - normally not too heavy to build and everything will be in your local maven.

**Comment by augusto-cactus** [comment]
No problem, just wanted to make you aware, I'll figure out something for our project. Thanks for your work 😃
